files/supervisord.conf

```diff
@@ -88,3 +88,18 @@ stderr_logfile=/var/log/supervisor/slurmd.log
 stderr_logfile_maxbytes=1MB
 stderr_logfile_backups=5
 priority=100
+
+[program:slurmrestd]
+user=root
+command=/usr/sbin/slurmrestd -vvvvv 0.0.0.0:6820
```

Comment:
Is this daemonized, or does it require the `-D` flag like the other slurm processes?

Reply:
Not, it's the foreground process, it will block